README.md

```diff
@@ -118,7 +118,7 @@ deny[{
 
 ### matches[[kind, namespace, name, matched_resource_output]]  
 
-While defining a `deny` rule finding resources that match a criteria is required, for instance if you are writing a violation for a specific ingress resource. kubernetes-policy-controller provides the matches funcationality by importing `data.kubernetes.matches`
+While defining a `deny` rule finding resources that match a criteria is required, for instance if you are writing a violation for a specific ingress resource. `kubernetes-policy-controller` provides the matches functionality by importing `data.kubernetes.matches`
```

Comment:
The first sentence is hard to parse. Instead:
> When defining a`deny` rule, you must find Kubernetes resources that match specific criteria, such as _Ingress_ resources in particular namespace.

Punctuate the second sentence.